.tributors

```diff
@@ -63,7 +63,8 @@
     "adelavega": {
         "name": "Alejandro de la Vega",
         "blog": "https://github.com/adelavega",
-        "email": "delavega@utexas.edu"
+        "email": "delavega@utexas.edu",
+        "bio": "Postdoc in the Psychoinformatics Lab"
```

Comment:
heh heh, tributors really wants a bio ATM... for me not to dig to fix it up somehow, Dear @adelavega , would you mind stating your desire for a `bio` entry? ;-)

Reply:
Haha It's fine. Where did it get that from? It's about out of date. Now "research scientist" ;)

Reply:
from https://github.com/adelavega ;)

Reply:
oh, haha!

Reply:
I updated it. if it changes good, if not, that's fine too

Reply:
I will close this one and see if new one is good